src/server_manager/web_app/app.ts

```diff
@@ -920,24 +926,41 @@ export class App {
         });
   }
 
-  private async setPortForNewAccessKeys(port: number, serverSettings: Polymer) {
+  private async setHostnameForAccessKeys(hostname: string, ui: Polymer) {
```

Comment:
What's `ui`? Could you give a more explicit name?

Reply:
Reverted to serverSettings